app/assets/stylesheets/partials/_flash_messages.scss

```diff
@@ -0,0 +1,16 @@
+.error-summary.wex_message {
```

Comment:
It's going to have a knock on effect throughout the PR, but can we rename this as `flash_message` or something else more generic?

It is highly likely we will want to use this same decision and functionality in WCR renewals, so dropping the service specific prefix will make it easier to port.

Reply:
Martin provided me with style and HTML for the messages. 
The main issue is that we are reusing style from the existing `error-summary` classes, which are defined in the Elements css, and give us responsiveness for free. This extra code here is just to adapt them to look as per screenshots. 
My suggestion is that we leave this as-it-is here and if this becomes an "important" bit later on before we move to the new GDS, then its worth spending more time tweaking this style.

Reply:
I'm fine with the styles, how it works etc. It's a great idea to reuse what we already have. I just don't want to use the prefix `wex` in this code.